order/models.py

```diff
@@ -0,0 +1,41 @@
+from tkinter import CASCADE
+from django.db import models
+from core import TimeStampModel
```

Comment:
- 첫 줄에 import 한 CASCADE는 잘못 import 한 것 같습니다. 제거해 주세요.
- 아래와 같이 package 종류에 따라 구분해 주세요.
```suggestion
from django.db import models

from core import TimeStampModel
```